src/utilities/color-transformers.ts

```diff
@@ -294,3 +305,34 @@ export function colorToHsla(color: string) {
       return color;
   }
 }
+
+export function normalizeValue(value: string) {
+  return value.toLowerCase().replace(/\s/g, '');
+}
+
+export function hexToHsb(hex: string) {
+  return rgbToHsb(hexToRgb(hex));
+}
+
+export function expandHex(hex: string) {
+  if (hex.length === 4) {
+    return `#${hex[1]}${hex[1]}${hex[2]}${hex[2]}${hex[3]}${hex[3]}`;
+  }
+  return hex;
+}
+
+export function nameToHex(value: string) {
+  const hex = nameHexMap[normalizeValue(value)];
+  return expandHex(`#${hex}`);
+}
+
+export function rgbStringToHex(value: string) {
+  const rgb = normalizeValue(value).match(
+    RGB_STRING_TO_HEX_REGEX,
+  ) as RegExpMatchArray;
```

Comment:
Hey @AndrewMusgrave I'm not sure what type to use here other than string, as we [use the function](https://github.com/Shopify/polaris-react/pull/2186/files/62d6e89075b848941a4bda8111e304f247cf76bd#diff-b3d0546d98cde51214257201a3fb16adR130) before we make any check if it is a valid colour entry.

Would you have any suggestions?

Reply:
Sorry I didn't realize we were chaing on a match. It looks like the return type is ` RegExpMatchArray | null` for match. It seems valid since match can return null. Rather than casting should we return a default value in case a match was not found?

Reply:
I added a default return value here if we don't match, but let me know if you think this is the wrong approach. Apologies if I'm misunderstanding your advice on this one.